presto-docs/src/main/sphinx/release/release-0.238.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Release 0.238
+=============
+
+**Highlights**
+==============
+* Fix case sensitivity issue in SQL function parameters. Function parameters should be treated case-insensitive.
+* Add support to create external function.
+* Add predicate pushdown support for ``DATE``, ``TIMESTAMP``, and ``TIMESTAMP_WITH_TIME_ZONE`` literals.
+
+General Changes
+_______________
+* Fix case sensitivity issue in SQL function parameters. Function parameters should be treated case-insensitive.
+* Add ``target_result_size`` session property to customize data batch sizes being streamed from coordinator.
+* Add optimization to push null filters to the INNER side of equijoins. The optimization can be enabled with ``optimize-nulls-in-joins``.
+* Add sessionProperty override to Presto JDBC URI.
+* Add support to create external function.
+* Remove max buffer count configuration property ``driver.max-page-partitioning-buffer-count`` for optimized repartitioning.
+
+Verifier Changes
+________________
+* Fix an issue where session properties of control and test queries also affects checksum queries.
+* Add support to report peak task memory usage for control and test queries.
+
+Geospatial Changes
+__________________
+* Fix error in geometry_invalid_reason.
+* Fix integer overflow in certain cases with Bing Tiles.
+
+Hive Changes
+____________
+* Fix a bug in Parquet reader which manifests when there are nested column schema changes.
+* Add support for exchange materialization of table bucketed by non-hive types. This can be enabled using value ``PRESTO_NATIVE`` for the ``bucket_function_type_for_exchange`` session property  or the ``hive.bucket-function-type-for-exchange `` configuration property.
```

Comment:
Following the convention in https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#phrasing - I will rework the second line as `This can be enabled by setting the abc session property or xyz configuration property to mno`